sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -321,6 +321,14 @@ enum class graph_state {
   executable
 };
 
+namespace property{ 
```

Comment:
```suggestion
namespace property { 
```

alternatively, could put this in the property namespace above we have for `property::node::depends_on`